content/docs/querying/api.md

```diff
@@ -0,0 +1,341 @@
+---
+title: HTTP API
+sort_rank: 7
+---
+
+# HTTP API
+
+The current stable HTTP API is reachable under `/api/v1` on a Prometheus
+server. Any non-breaking additions will be added under that endpoint. There are
+no minor versions of the API.
```

Comment:
This was more for the design doc. I don't think we have to document what we not have.